src/explore-education-statistics-frontend/src/modules/find-statistics/components/PublicationSectionBlocks.tsx

```diff
@@ -3,25 +3,24 @@ import useGetReleaseFile from '@common/modules/release/hooks/useGetReleaseFile';
 import ContentBlockRenderer from '@common/modules/find-statistics/components/ContentBlockRenderer';
 import DataBlockTabs from '@common/modules/find-statistics/components/DataBlockTabs';
 import useReleaseImageAttributeTransformer from '@common/modules/release/hooks/useReleaseImageAttributeTransformer';
-import { Release } from '@common/services/publicationService';
 import { Block } from '@common/services/types/blocks';
 import ButtonLink from '@frontend/components/ButtonLink';
 import { logEvent } from '@frontend/services/googleAnalyticsService';
 import React from 'react';
 
 export interface PublicationSectionBlocksProps {
-  release: Release;
+  releaseId: string;
```

Comment:
I don't think this change is related to any problems found, and it might have been safer to keep it as Release so that it's known to definitely be the ReleaseId rather than just any other string? It might be useful for the future too if other fields of Release are needed.

Reply:
I figured that since only the releaseId was being used that there was no reason to pass the whole release. Just did it I think when I was adding the `replace('/api', '')` stuff. I'll change it back

Reply:
Ok, cool 🙂